### PR TITLE
feat: file, chat and contact cache

### PR DESCRIPTION
**Pull request by anri-asaturov**

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -57,5 +57,26 @@ function tryToGet(fn, defaultValue) {
     return defaultValue;
 }
 
-module.exports = { convertBuffers, formatBytes, tryToGet };
+
+function getCacheDbFullName(name) {
+    if (!name) throw new Error('Cache database has to have a name');
+    const prefix = 'peerio'; // something to separate our databases in case they're in global scope
```
Comment (seavan) [inline src/util.js]: should we add applabel support here?

File: src/helpers/performance-polyfill.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+(function(scope) {
```
Comment (seavan) [inline src/helpers/performance-polyfill.js]: it would be nice to have performance features reviewed and merged in a separate branch.

Comment (anri-asaturov) [reply src/helpers/performance-polyfill.js]: What performance features?

File: src/util.js (hunk 1 above)
Comment (anri-asaturov) [reply src/util.js]: database is per-app right? so app label is not needed

**Comment by ebilg** [comment]
can be merged

**Pull request merged**